api.scm

```diff
@@ -83,6 +83,16 @@
 ; check name property of music object ... just a shortcut
 (define-public (music-is? m n) (and (ly:music? m)(eq? n (ly:music-property m 'name))))
 
+(define-public (in-list? val ls)
+  (if (null? ls) #f
+      (if (equal? val (car ls)) #t (in-list? val (cdr ls)))))
+
+(define (intersection ls1 ls2)
+  (if (null? ls1) '()
+      (if (in-list? (car ls1) ls2)
+          (cons (car ls1) (intersection (cdr ls1) ls2))
+          (intersection (cdr ls1) ls2))))
```

Comment:
Changes look good, thanks @rshura !  Here we could just use Guile's `lset-intersection` rather than defining our own.  See: https://www.gnu.org/software/guile/docs/docs-1.8/guile-ref/SRFI_002d1-Set-Operations.html#index-lset_002dintersection-3662

Reply:
Tried that, got this:
`/home/shura/src/lilypond-export/api.scm:358:33: Unbound variable: lset-intersection`

Do I need to import some module before?

Reply:
Added https://github.com/openlilylib/lilypond-export/pull/23 to move to the existing set functions.

Reply:
Thanks @rshura !  I wasn't sure if the module needed to be imported or not.  I tried using the function in Frescobaldi and it worked there in a .ly file, so it looks like LilyPond makes that module available.  But good to know it needs to be imported in .scm files.